front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -165,8 +165,10 @@ export class AgentTablesQueryAction extends WorkspaceAwareModel<AgentTablesQuery
 
   declare step: number;
   declare resultsFileId: ForeignKey<FileModel["id"]> | null;
+  declare searchableFileId: ForeignKey<FileModel["id"]> | null;
```

Comment:
Last nit, searchableFile might not be the best name :grimacing: dustSectionFileId or sectionFileId ? :grimacing: :grimacing: :grimacing:

Reply:
renamed everything to `sectionFileId`!